jbmc/src/jbmc/bmc.cpp

```diff
@@ -204,7 +204,7 @@ safety_checkert::resultt bmct::stop_on_fail()
     if(options.get_bool_option("trace"))
     {
       if(options.get_bool_option("beautify"))
-        counterexample_beautificationt()(
+        counterexample_beautificationt{ui_message_handler}(
```

Comment:
`{}` is not just for beauty here - `()` is considered ambiguous and won't compile.

Reply:
Fair, but the coding guidelines also suggest it should be used everywhere...